test/DebugInfo/comp-dir.swift

```diff
@@ -0,0 +1,27 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-swift-frontend -c -primary-file %s -g -parse-as-library -emit-module -emit-object -module-cache-path "./mc" -file-compilation-dir "." -debug-prefix-map "$(pwd)=." -o %t/test.o
+// RUN: %llvm-dwarfdump %t/test.o | %FileCheck %s
+
+// CHECK: DW_TAG_compile_unit
+// CHECK: DW_TAG_compile_unit
+// CHECK-NEXT: DW_AT_producer	("Swift version
+// CHECK-NEXT: DW_AT_language	(DW_LANG_ObjC)
+// CHECK-NEXT: DW_AT_name	("Foundation")
+// CHECK-NEXT: DW_AT_stmt_list	(0x{{[0-9a-z]+}})
+// CHECK-NEXT: DW_AT_comp_dir	(".")
+// CHECK-NEXT: DW_AT_APPLE_optimized	(true)
+// CHECK-NEXT: DW_AT_GNU_dwo_id	(0x{{[0-9a-z]+}})
+// CHECK-NEXT: DW_AT_GNU_dwo_name	("./mc/{{.*}})
+
+// CHECK: DW_TAG_compile_unit
+// CHECK-NEXT: DW_AT_producer	("Swift version
+// CHECK-NEXT: DW_AT_language	(DW_LANG_ObjC)
```

Comment:
This might not match - not all targets use ObjC. I would suggest that you pass `-disable-objc-interop` so that the behaviour is uniform on all the platforms and this is likely going to be marked as `DW_LANG_C`.